config/routes.rb

```diff
@@ -118,6 +118,7 @@ def scoped_navigation_routes(context, navigation)
       end
       get "/diy", to: redirect { Diy::FileYourselfController.to_path_helper }
       get "/diy/email", to: redirect { Diy::FileYourselfController.to_path_helper }
+      get "/click_fsa_link", action: :click_fsa_link, controller: Diy::ContinueToFsaController.controller_path, as: :click_fsa_link
```

Comment:
nest under /diy prolly

Reply:
Rather than controller & action combo I prefer `signups#confirmation` style but we can see